contracts/verifiers/FflonkVerifier.sol

```diff
@@ -48,8 +48,8 @@ contract FflonkVerifier {
     uint256 constant w8_7 = 8613538655231327379234925296132678673308827349856085326283699237864372525723;
 
     // Verifier preprocessed input C_0(x)·[1]_1
-    uint256 constant C0x  = 10330861150616913541207360924312278787207684679419670167996002791305440821704;
-    uint256 constant C0y  = 14244524222232642134204580826860575549922689127759263461569792153452315036880;
+    uint256 constant C0x  = 7436841426934271843999872946312645822871802402068881571108027575346498207286;
```

Comment:
No idea what this is but do we mean to change it?

Reply:
Yes, the new verifier needs an update for the numbers. 

It's because the circuits of the system changed